OPUNetTransportLayer.cpp

```diff
@@ -1114,7 +1114,7 @@ bool OPUNetTransportLayer::DoImmediateProcessing(Packet &packet, sockaddr_in &fr
 				tlMessage.tlHeader.commandType = tlcJoinGranted;
 				// **DEBUG**
 				Log("Client join accepted: " + FormatAddress(fromAddress) + 
-					" (" + std::to_string(tlMessage.joinReply.newPlayerNetID) + ")");
+					FormatPlayerNetID(tlMessage.joinReply.newPlayerNetID));

```

Comment:
This might need some spacing between the address and the playerNetId.

Reply:
Tried to add some spacing. Please suggest how you want it to read if this doesn't look good. Better than me guessing what you are thinking.

Reply:
I noticed that the space between the IP address and the playerNetId had been removed. (Assuming I read the code correctly). This seems to resolve any reading difficulties.

Reply:
Sounds good.